Tests/Mocks/MockStorageContainer.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MockStorageContainer.swift
+//  SessionTools-iOSTests
+//
+//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+import SessionTools
+
+/// An in memory container for testing.
+public class MockStorageContainer<T: Codable>: SessionContainer {
```

Comment:
Since this is in the test target, I don't think we need it to be `public` (same applies to its methods).

Reply:
Agreed. Removed public.